recipes/gstreamer/all/conanfile.py

```diff
@@ -1,18 +1,29 @@
-from conans import ConanFile, tools, Meson, VisualStudioBuildEnvironment
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.tools.build import cross_building
+from conan.tools.env import VirtualBuildEnv, VirtualRunEnv
+from conan.tools.files import chdir, copy, get, rename, rm, rmdir
+from conan.tools.gnu import PkgConfigDeps
+from conan.tools.layout import basic_layout
+from conan.tools.meson import Meson, MesonToolchain
+from conan.tools.microsoft import is_msvc
+from conan.tools.scm import Version
+
 import glob
 import os
-import shutil
 
-required_conan_version = ">=1.29"
+required_conan_version = ">=1.56"
+
 
 class GStreamerConan(ConanFile):
     name = "gstreamer"
     description = "GStreamer is a development framework for creating applications like media players, video editors, streaming media broadcasters and so on"
-    topics = ("conan", "gstreamer", "multimedia", "video", "audio", "broadcasting", "framework", "media")
+    topics = ("multimedia", "video", "audio",
+              "broadcasting", "framework", "media")
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://gstreamer.freedesktop.org/"
-    license = "GPL-2.0-only"
+    license = "LGPL-2.0-or-later"
```

Comment:
Why was this changed?

Reply:
I did due to the other PR did it 😄 
But reading [this](https://gstreamer.freedesktop.org/documentation/frequently-asked-questions/licensing.html?gi-language=c)  it should be correct to change it to LGPL.